### DISCUSSION TITLE
The optimal number of trials for Bayesian optimization

### DISCUSSION THREAD

**Discussion by MariaBocsa** [optuna/optuna#4011] [Q&A] [answered]
Hello,
I am implementing Optuna search using the LightGBM model for a regression problem with time series data and I want to know how many trials to use to optimize the hyperparameters. 
Is there a rule to find the optimal number of trials for Bayesian Optimization method?

**Comment by not522** [comment, marked as answer]
The optimal number of trials depends on the problem, but the recommendation is listed in the samplers comparison table. https://optuna.readthedocs.io/en/stable/reference/samplers/index.html
